uint8  Control_Mode_flag                                                 
bool CAN_Start
bool CAN_Stop       
int8 Acc_Value
int16 Angle_Value